server/src/main/java/com/jungleapp/cs414/server/HTTPRestful.java

```diff
@@ -54,17 +54,19 @@ private boolean login(Request request, Response response) {
         return loginProfile.establishProfileIdentity();
     }
 
-    private String board(Request request, Response response) {
```

Comment:
"/startGame" and "/move" returns the exact same code. Would it be better to revise the name of each method and instead make a single get request which makes sense in both of those situations?